ccp/modules/dnpm-node-compose.yml

```diff
@@ -1,34 +1,88 @@
 version: "3.7"
 
 services:
-  dnpm-backend:
-    image: ghcr.io/kohlbacherlab/bwhc-backend:1.0-snapshot-broker-connector
-    container_name: bridgehead-dnpm-backend
+  dnpm-mysql:
+    image: mysql:latest
```

Comment:
Consider pinning this at least to a major version

Reply:
Done in [7c613f4](https://github.com/samply/bridgehead/pull/251/commits/7c613f483abb799149cb3430860ad6b66475a2ab)